R/render-cached-plot.R

```diff
@@ -398,6 +400,8 @@ renderCachedPlot <- function(expr,
             func = isolatedFunc,
             width = width,
             height = height,
+#jy: default to title when NULL
+            alt = alt,
```

Comment:
Set `alt <- altWrapper()` about 15 lines up where `height` and `width` are set inside the `isolate()` call.